lib/src/database/database_api.dart

```diff
@@ -82,7 +82,7 @@ abstract class DatabaseApi {
 
   Future<List<User>> getUsers(Room room);
 
-  Future<void> storeUsers(List<User> users, Room room);
```

Comment:
why not keep this API? IMO, store a batch of user still useful

Reply:
Reverted code and wrote new test case for this.